problemsets-mat223/modules/module12.tex

```diff
@@ -85,8 +85,18 @@
 \begin{example}
 	Let $\mathcal P:\R^2\to\R^2$ be projection onto the $x$-axis and let $\mathcal R:\R^2\to\R^2$ be rotation counter-clockwise
 	by $15^\circ$. Classify each of $\mathcal P$ and $\mathcal R$ as invertible or not.
-
-	XXX Finish
+	
+	Since vectors on y-axis projected onto the x-axis are $\vec 0$, we know
+	\[
+	    \Null(\mathcal P)=\Set*{\vec v\in\R:\vec v=k\mat{0\\1}\text{, for some k}\in\R^2}.
```

Comment:
Use `\given` instead of `:` in sets. Don't put a `,` before `for some`. The `k` should not be inside the `\text{...}`